tests/test_mesh.py

```diff
@@ -294,6 +298,35 @@ def test_usd_parse(usd_filename):
         )
 
 
+@pytest.mark.skipif(
+    sys.version_info[:2] != (3, 10) or sys.platform not in ("Linux", "Windows"),
+    reason="USD Baking only works in Python == 3.10 and OS == ('Linux', 'Windows') now."
```

Comment:
You should rather mention what is the technical limitation. The reason you mentioned is already pretty obvious for anybody reading the code.